1.6/Dockerfile.rhel7

```diff
@@ -36,19 +36,18 @@ EXPOSE 8080 50000
 #RUN  curl -L -o /tmp/ssh-credentials.hpi http://updates.jenkins-ci.org/latest/ssh-credentials.hpi
 #RUN  curl -L -o /tmp/credentials.hpi http://updates.jenkins-ci.org/latest/credentials.hpi
 
-RUN yum install -y tar zip unzip java-1.7.0-openjdk jenkins-1.609.1 && \
+COPY run.sh /usr/local/bin/run.sh
+COPY ./contrib/ /opt/openshift
+
+RUN yum install -y tar zip unzip java-1.7.0-openjdk jenkins-1.609 && \
     yum clean all  && \
     chmod og+rw /var/lib/jenkins && \
+    chmod -R og+rw /opt/openshift && \
```

Comment:
nit: Maybe keeping this chmod line on the same line as in the CentOS Dockerfile for consistency?